source/about.rst

```diff
@@ -113,6 +113,32 @@ may go by different names.
     .. [#3] JavaScript does not have separate types for integer and
             floating-point.
 
+    --PHP
+    The following table maps from the names of JavaScript types to
+    their analogous types in PHP:
+
+    +----------+--------------+
+    |JavaScript|PHP           |
+    +----------+--------------+
+    |string    |String        |
+    +----------+--------------+
+    |number    |Integer/Float |
+    |          |[#4]_         |
+    +----------+--------------+
+    |object    |Object        |
+    +----------+--------------+
+    |array     |Array         |
+    +----------+--------------+
+    |boolean   |Boolean       |
+    +----------+--------------+
+    |null      |NULL          |
+    +----------+--------------+
```

Comment:
I'm not really sure whether PHP types should be spelled in upper-case or lower-case.  I don't think it really matters since the built-in types are not first-class objects, though `gettype()` returns the lower-case spelling.  Either way I think one spelling should be used consistently (and in some of the examples below you use all lower-case instead).